lib/manager/npm/update/dependency/index.ts

```diff
@@ -3,6 +3,55 @@ import { logger } from '../../../../logger';
 import { matchAt, replaceAt } from '../../../../util/string';
 import type { UpdateDependencyConfig } from '../../../types';
 
+function replaceAsString(
```

Comment:
Let's merge this as a refactor PR first, once we're happy with it

Reply:
#11195